src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {
+            "червоний", "зелений", "синій", "жовтий", "чорний"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
Consider moving the 'Random' instance to a class-level variable. This way, you won't need to instantiate a new Random object every time the 'getRandomColor' method is called.

Reply:
good advice